package.json

```diff
@@ -90,6 +90,7 @@
     "axios": "^1.9.0",
     "bignumber.js": "^9.1.2",
     "bs58": "^5.0.0",
+    "cli-progress": "^3.12.0",
```

Comment:
this should be excluded from web builds in similar ways as `commander`

Reply:
Excluded.